tools/font-edit/README.md

```diff
@@ -14,4 +14,13 @@ make
 ./twin-fedit < nchars
 ```
 
-(press 'q' to next character)
+| Key binding | Function |
+|   ---       |   ---    |
+| `esc`       | Exit program |
+| `left mouse button` | Select a point as start point |
+| `right mouse button` | Select a point as end point |
+| `d` | Delete selected point|
```

Comment:
Explain start/end points before key binding. You should mention the background knowledge prior to key bindings.